packages/core/src/App/AppContent.tsx

```diff
@@ -31,6 +38,37 @@ const AppContent: React.FC<{ passthrough: unknown }> = observer(({ passthrough }
     const { data } = useRemoteConfig(isMounted());
     const { marketing_growthbook, tracking_datadog, tracking_rudderstack } = data;
 
+    const account_type = LocalStore?.get('active_loginid')
+        ?.match(/[a-zA-Z]+/g)
+        ?.join('');
+
+    const attributes = {
+        account_type: account_type === 'null' ? 'unlogged' : account_type,
+        app_id: String(getAppId()),
+        device_type: store?.ui?.is_mobile ? 'mobile' : 'desktop',
+        device_language: navigator?.language || 'en-EN',
+        user_language: getLanguage().toLowerCase(),
+        country: Cookies.get('clients_country') || Cookies?.getJSON('website_status'),
+    };
```

Comment:
[Future Improvement] We need to abstract out these info from the appcontent file.

Reply:
yes, agree. 
for my opinion, we could add provider for the whole app for using analytics, and move all configs in separate files